huaweicloud/services/rds/resource_huaweicloud_rds_account.go

```diff
@@ -0,0 +1,184 @@
+		UpdateContext: resourceRdsAccountUpdate,
+		DeleteContext: resourceRdsAccountDelete,
+		ReadContext:   resourceRdsAccountRead,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"instance_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"password": {
+				Type:      schema.TypeString,
+				Required:  true,
+				Sensitive: true,
+			},
+		},
+	}
+}
+
+func resourceRdsAccountCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	c := meta.(*config.Config)
+	client, err := c.HcRdsV3Client(c.GetRegion(d))
+	if err != nil {
+		return fmtp.DiagErrorf("error creating RDS client: %s", err)
+	}
+
+	dbUser := d.Get("name").(string)
+	instanceId := d.Get("instance_id").(string)
+	createOpts := &model.CreateDbUserRequest{
+		InstanceId: instanceId,
+		Body: &model.UserForCreation{
+			Name:     dbUser,
+			Password: d.Get("password").(string),
+		},
+	}
+
+	_, err = client.CreateDbUser(createOpts)
+	if err != nil {
+		return fmtp.DiagErrorf("error creating RDS db user: %s", err)
+	}
+
+	id := instanceId + "/" + dbUser
+	d.SetId(id)
+	return resourceRdsAccountRead(ctx, d, meta)
+}
+
+func resourceRdsAccountRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	c := meta.(*config.Config)
+	client, err := c.HcRdsV3Client(c.GetRegion(d))
+	if err != nil {
+		return fmtp.DiagErrorf("error creating RDS client: %s", err)
+	}
+
+	// Split instance_id and user from resource id
+	parts := strings.SplitN(d.Id(), "/", 2)
+	if len(parts) != 2 {
+		return fmtp.DiagErrorf("invalid id format, must be <instance_id>/<user>")
+	}
+	instanceId := parts[0]
+	dbUser := parts[1]
+
+	// items on every page, [1, 100]
+	limit := int32(100)
+	// List all db users
+	request := &model.ListDbUsersRequest{
+		InstanceId: instanceId,
+		Limit:      limit,
+		Page:       int32(1),
+	}
+
+	for {
+		response, err := client.ListDbUsers(request)
+		if err != nil {
+			return common.CheckDeletedError(d, err, "error listing RDS db users")
+		}
+		users := *response.Users
+		if len(users) == 0 {
+			break
+		} else {
+			request.Page += 1
+			for _, user := range users {
+				if user.Name == dbUser {
+					d.Set("instance_id", instanceId)
+					d.Set("name", dbUser)
+					return nil
+				}
+			}
+		}
+	}
+
+	// DB user deleted
+	d.SetId("")
+	log.Printf("[WARN] failed to fetch RDS db user %s: deleted", dbUser)
+
+	return nil
+}
+
+func resourceRdsAccountUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	c := meta.(*config.Config)
+	client, err := c.HcRdsV3Client(c.GetRegion(d))
+	if err != nil {
+		return fmtp.DiagErrorf("error creating RDS client: %s", err)
+	}
+
+	// Split instance_id and user from resource id
+	parts := strings.SplitN(d.Id(), "/", 2)
```

Comment:
it seems that we can get instanceID and  dbuser from d.Get

Reply:
done